### ISSUE TITLE
bug: can't apply custom css

### ISSUE THREAD

**Issue by herrera-ignacio** [simonhaenisch/md-to-pdf#89]
**Context:**

- version: 3.2.1
- platform: macOS
- node version: 14.16.0

**Describe the bug:**

Cannot apply custom css. Every other option seems to be working properly, I have the following code snippet working fine except for the css option that doesn't have any visible effect.

```javascript
    return mdToPdf({ content: data }, {
      pdf_options: {
        format: "a4",
        margin: "20mm",
        printBackground: true,
        displayHeaderFooter: true,
        footerTemplate: `
          <style>
            section {
              margin: 0 auto;
              font-size: 10px;
            }
          </style>
          <section>
            <div>
              Page <span class="pageNumber"></span>
              of <span class="totalPages"></span>
            </div>
          </section>`
      },
      css: `body { color: red; }`,
      launch_options: {
        args: ["--no-sandbox"],
      }
```

I'd expect that all body's text elements have red color, but none of them do. Any ideas?

**Comment by simonhaenisch** [comment]
It works if you do `body { color: red !important }` but i'm not sure why the `!important` is needed 🤔 

Adding the stylesheets into the page is async and done in parallel, so maybe it's just a problem with order of execution (i. e. your custom css is injected before the default stylesheet). I'll look into it.

**Comment by simonhaenisch** [comment]
Yeah I can confirm that this is a problem with the order of how the scripts are injected.

I'll commit and release a fix soon.

**Comment by simonhaenisch** [comment]
e1b6003a1f1e2ad512706215ab5d0ddb992c9f00

**Comment by simonhaenisch** [comment]
Just released 3.3.2 which should have fixed this. Let me know/reopen if it didn't (:
